.github/workflows/python.yml

```diff
@@ -119,18 +119,50 @@ jobs:
           cache-workspace-crates: true
       - uses: ./.github/workflows/build_linux_wheel
       - uses: ./.github/workflows/run_tests
+      - name: Upload wheels as artifacts
+        if: ${{ matrix.python-minor-version == '12' }}
+        uses: actions/upload-artifact@v4
+        with:
+          name: linux-wheels
+          path: python/target/wheels/pylance-*.whl
+
+  forward-compat:
+    needs: linux
+    runs-on: ubuntu-24.04
+    name: Forward Compatibility Tests (${{ matrix.lance-version }})
+    strategy:
+      matrix:
+        lance-version: ["0.16.0", "0.29.1.beta2", "0.36.0"]
```

Comment:
how will this be kept up to date?

Reply:
I was wondering if maybe it would be an improvement to always test compatibility relationship with the previous release or n-2 or whatever. That also won't test the full matrix but it would stay up to date without maintenance. Do you think that would be any better?

Reply:
+1 on having a recent one.